commitizen/providers.py

```diff
@@ -115,14 +115,25 @@ def set(self, pyproject: tomlkit.TOMLDocument, version: str):
 class CargoProvider(TomlProvider):
     """
     Cargo version management
+
+    With support for `workspaces`
     """
 
     filename = "Cargo.toml"
 
     def get(self, document: tomlkit.TOMLDocument) -> str:
-        return document["package"]["version"]  # type: ignore
+        try:
+            return document["package"]["version"]  # type: ignore
```

Comment:
would it be possible for us to try `document["package"].get("version")`? IMO, it's more readible

Reply:
The idea is to make it fail if any of the keys are not present

Reply:
got it. Make sense 👍